lib/mutant/ast/regexp/transformer.rb

```diff
@@ -0,0 +1,82 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        include AbstractType
+
+        REGISTRY = {} # rubocop:disable MutableConstant
```

Comment:
Instead of a constant, what about using `@registry = {}` here? Then there's no question it's mutable.

Also, I assume that after some initial population of the registry, it's contents don't change again? If that's so, then I might suggest adding a `.finalize` class method that deep freezes the registry so it can't be changed at runtime.